### ISSUE TITLE
Cluster label includes 'Cluster [...]' instead of just the cluster name

### ISSUE THREAD

**Issue by JorritSalverda** [vvanholl/elasticsearch-prometheus-exporter#13]
When upgrading from 2.4.1 to 5.0.1 the cluster label added some static wording that's really redundant:

```
es_jvm_gc_collection_time_seconds{cluster="Cluster [elasticsearch]",node="10.248.8.97",gc="young",} 0.0
```

I would expect just `cluster="elasticsearch"` instead.

**Comment by rosskukulinski** [comment]
I'm also seeing this and was thrown off!

**Comment by vvanholl** [comment]
@rosskukulinski : This problem should be solved from many weeks now. What version of the plugin do you use ?

**Comment by rosskukulinski** [comment]
@vvanholl ahh - I'm still on 5.3.0.  I see it's fixed in 5.3.1.  Thanks
